lib/v2/screens/sign_up/create_username/create_username.dart

```diff
@@ -0,0 +1,100 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:seeds/i18n/edit_name.i18n.dart';
+import 'package:seeds/v2/components/flat_button_long.dart';
+import 'package:seeds/v2/components/quadstate_clipboard_icon_button.dart';
+import 'package:seeds/v2/components/text_form_field_custom.dart';
+import 'package:seeds/v2/design/app_theme.dart';
+import 'package:seeds/v2/domain-shared/page_state.dart';
+import 'package:seeds/v2/screens/sign_up/viewmodels/bloc.dart';
+import 'package:seeds/v2/utils/debouncer.dart';
+
+class CreateUsername extends StatefulWidget {
```

Comment:
CreateUsername const constructor missing

Reply:
Sorry, I'm not following. Why do we need to have a constructor here?

Reply:
If you build your own widgets or use Flutter widgets this helps Flutter to rebuild only widgets that should be updated.

const reduces the required work for the garbage collector, minimize memory allocation, also it allows you to define such items only once during AOT compilation. 

It's definetly not improve your app performance significantly, but using const will make it a little bit faster.